tests/core/Resolver.spec.ts

```diff
@@ -110,13 +108,12 @@ describe('Resolver', () => {
       const newServiceEndpoints = OperationGenerator.generateServiceEndpoints(['newDummyHubUri1']);
 
       // Create the recover operation and insert it to the operation store.
-      const [update1RevealValueAfterRecovery, update1CommitmentHashAfterRecovery] = OperationGenerator.generateCommitRevealPair();
       const recoverOperationJson = await OperationGenerator.generateRecoverOperationRequest(
         didUniqueSuffix,
         recoveryPrivateKey,
         newRecoveryPublicKey,
         newSigningPublicKey,
-        update1CommitmentHashAfterRecovery,
+        Multihash.canonicalizeThenHashThenEncode(newSigningPublicKey.jwk),
```

Comment:
Sanity: this parameter should be removed since `newSigningPublicKey` is passed in, this should be computed internally.